libraries/AP_Math/vector2.cpp

```diff
@@ -126,11 +131,11 @@ bool Vector2<T>::operator !=(const Vector2<T> &v) const
 template <typename T>
 float Vector2<T>::angle(const Vector2<T> &v2) const
 {
-    const float len = this->length() * v2.length();
+    float len = this->length() * v2.length();
```

Comment:
Why are you removing const?

Reply:
I suspect this was a merge conflict as I pulled in some commits from the OBC branch.  restored!